tools/clang/test/CodeGenHLSL/overload_new_and_delete_operators.hlsl

```diff
@@ -0,0 +1,25 @@
+// RUN: %dxc -E main -T vs_6_0 -enable-operator-overloading %s | FileCheck %s
+
+// This test checks that when we overload new or delete operator
+// dxcompiler generates error and no crashes are observed.
+
+// CHECK: error: overloading 'operator new' is not allowed
+// CHECK: error: overloading 'operator delete' is not allowed
+// CHECK: error: 'new' is a reserved keyword in HLSL
+// CHECK: error: 'delete' is a reserved keyword in HLSL
+
```

Comment:
Move it to VerifierTest

Reply:
@pow2clk @tex3d 

This test checks if it correctly reports errors for `new` and `delete` even when `-enable-operator-overloading` is enabled.
I checked unit tests in `VerifierTest` and found out that they use `%clang_cc1` e.g., `%clang_cc1 -fsyntax-only -Wno-unused-value -ffreestanding -verify %s`.
I am not sure there is a way to enable `-enable-operator-overloading` for `clang_cc1`.
I would appreciate any advice!

Reply:
My expectation is that you'll be able to provide this argument to `clang_cc1`. In spite of the name, it's not really a clang compiler, but a keyword that is parsed by code in `VerifierTest.cpp` and `CompilationResult.h`. 

Did you try it and it failed?

Reply:
Oh I did not try it. It seems to work. Thanks!